src/common/PersonTile/index.js

```diff
@@ -6,7 +6,7 @@ export const PersonTile = ({name, role, portrait, id}) => {
 
     return (
         <Content onClick={() => navigate(`../people/${id}`)}>
-            <Img src={portrait === "https://image.tmdb.org/t/p/w780/null" ? null : portrait} alt={name} loading="lazy" />
+            <Img src={portrait === "https://image.tmdb.org/t/p/w500/null" ? null : portrait} alt={name} loading="lazy" />
```

Comment:
why changing it to 500 ?

Reply:
It loads faster, we dont need w780, it is too big and not changes much.